### ISSUE TITLE
Ping is always shown as compatible when client and server version are the same

### ISSUE THREAD

**Issue by Madis0** [ViaVersion/ViaFabric#68]
**Describe the bug, provide any errors**
The server list player counts still depend on the real client, instead of the ViaFabric-emulated version

**Is this happening specifically on ViaFabric?**
(Optional)
- [x] It seems to not happen when using ViaVersion/ViaBackwards/ViaRewind installed on Bukkit/BungeeCord/Sponge/Velocity server.

**How can we reproduce it?**
1. Run Minecraft 1.16.3 with ViaFabric
2. Open Multiplayer and see how the servers are displayed (which are compatible and which not)
3. Click the VIA button and set the version to something different
4. Note the servers again - it is identical, no matter what version is set in ViaFabric

**Expected behaviour**
Assuming the server doesn't have Via* installed, the servers should be marked as incompatible (version number in red)

**Screenshots**
N/A

**ViaVersion Dump:**
Version numbers are as follows, the rest seems to be irrelevant in this bug.
```
    "versionInfo": {
        "javaVersion": "14.0.2",
        "operatingSystem": "Linux",
        "serverProtocol": 753,
        "enabledProtocols": [
            753
        ],
        "platformName": "ViaFabric",
        "platformVersion": "0.2.15-SNAPSHOT+build-568.1-mc-1.16",
        "pluginVersion": "3.1.1",
        "subPlatforms": []
    },
```

**Additional server info**
N/A

**Comment by Username404-59** [comment]
I don't understand why you want this feature, you already did [an issue for multiconnect](https://github.com/Earthcomputer/multiconnect/issues/115) and now you want it to be implemented in ViaFabric ?
If you want this feature so hard why don't you just make a mod to add it?

**Comment by Madis0** [comment]
> I don't understand why you want this feature, you already did an issue for multiconnect for something like that but also want it to be implemented in ViaFabric ?
Since the mods are similar, I suggested the same for both, yes. Why - for giving more information to the user, so they know how the server works (e.g. 1.8 mechanics) and who can they invite to it.

> If you really want it why don't you just make a mod to add it (if you know some Java, making tiny mods for fabric is easy)?
I do know Java, but I haven't yet figured out how exactly Minecraft mod development works, hopefully someday I will make my own mods.

> Edit: What's the point of marking servers as incompatibles if ViaFabric works with them?
But it doesn't if the user sets the version to an incompatible one, that's the point.

> Edit 2: java 14 is outdated btw, the latest version is 15 .
Thanks.

**Comment by Earthcomputer** [comment]
Java 14 is the latest LTS Java version, so it still makes sense to use it.

**Comment by BoogieMonster1O1** [comment]
> Java 14 is the latest LTS Java version, so it still makes sense to use it.

java 14 isnt lts

**Comment by Earthcomputer** [comment]
Oops

**Comment by Username404-59** [comment]
> But it doesn't if the user sets the version to an incompatible one, that's the point.
Well, ViaFabric auto-detects the version of the server when you write **AUTO** in the version box, so you don't have to change the version for each server

**Comment by creeper123123321** [comment]
Related to  #35

**Comment by Madis0** [comment]
Mostly fixed in ViaFabric 0.2.16-SNAPSHOT+build-586.1/Minecraft 1.16.4, only broken in vanilla servers.

**Comment by Kichura** [comment]
Not reproducable, This might be fixed in current version of viafabric - if it persists, make a reply here.
